lib/src/test/java/com/dynatrace/metric/util/NormalizeTest.java

```diff
@@ -202,4 +226,14 @@ private static String createNonsenseStringOfLength(int n) {
             createNonsenseStringOfLength(270),
             createNonsenseStringOfLength(250)));
   }
+
+  private static Stream<Arguments> provideToEscapeValues() {
```

Comment:
Do we also need to escape double quotes (`"`)?

Reply:
quotes would only have to be escaped if we put the dimension values in quotes before sending them